ReSharper.FSharp/src/FSharp.Psi.Features/src/Intentions/FunctionArgumentTypesAnnotationAction.fs

```diff
@@ -0,0 +1,21 @@
+﻿namespace JetBrains.ReSharper.Plugins.FSharp.Psi.Features.Intentions
+
+open JetBrains.ReSharper.Feature.Services.ContextActions
+open JetBrains.ReSharper.Plugins.FSharp.Psi.Impl
+open JetBrains.ReSharper.Plugins.FSharp.Psi.Tree
+
+[<ContextAction(Name = "AnnotateFunctionArgumentTypes", Group = "F#",
+                Description = "Annotate function with argument types")>]
```

Comment:
A parameter is a variable in a method definition. When a method is called, the arguments are the data you pass into the method's parameters.

```suggestion
                Description = "Annotate function parameter types")>]
```